src/index.js

```diff
@@ -0,0 +1,77 @@
+import './style.css';
+
+const input = document.querySelector('#new-task-input');
+const submitButton = document.querySelector('.add');
+const taskList = document.querySelector('#tasks');
+let tasks = [];
+
+if (localStorage.getItem('tasks')) {
+  tasks = JSON.parse(localStorage.getItem('tasks'));
+}
+
+/*eslint-disable*/
+loadTasks();
+
+// Add task
+submitButton.onclick = function (e) {
+  e.preventDefault();
+  if (input.value !== "") {
+    /*eslint-disable*/
+    createNewTask(input.value);
+    input.value = "";
+  }
+};
```

Comment:
- [ ] According to the requirements:
![image](https://user-images.githubusercontent.com/77069011/222309458-83108e27-828a-498a-bc8e-9c17910b09d4.png)

This new functionality should be in a new, separate file, please consider applying it! 🤓